app/presenters/search_result_presenter.rb

```diff
@@ -8,29 +10,76 @@ class SearchResultPresenter
            :es_score,
            to: :search_result
 
-  def initialize(search_result, metadata)
-    @search_result = search_result
-    @metadata = metadata
+  def initialize(data = {})
+    @search_result = data[:search_result]
+    @metadata = data[:metadata]
+    @index = data[:doc_index] + 1
+    @count = data[:doc_count]
+    @finder_name = data[:finder_name]
+    @debug_score = data[:debug_score]
+    @highlight = data[:highlight]
   end
 
   def to_hash
```

Comment:
Maybe we could rename this to govuk_component_hash or something to indicate that this hash is to be used directly in the component

Reply:
Is that important? Is `to_hash` generally reserved for something else?

Reply:
Renamed.